hack/verify-godeps.sh

```diff
@@ -88,7 +88,7 @@ rm -rf ./Godeps ./vendor
 git init > /dev/null 2>&1
 
 # Recreate the Godeps using the nice clean set we just downloaded
-"${GODEP}" save ./...
+hack/godep-save.sh
```

Comment:
also change docs/devel/development.md to document calling this script rather that